common/components/card/template.njk

```diff
@@ -68,5 +69,15 @@
     {% if params.insetText %}
       {{ govukInsetText(params.insetText) }}
     {% endif %}
+
+    {% if params.isLockout %}
+      <div class="card-warning-text">
+        {{ govukWarningText({
+          text: "Prison lockout",
```

Comment:
I think we should move this text in to a [locales](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/tree/main/locales/en) file.